inputfiles/overridingTypes.json

```diff
@@ -338,6 +338,7 @@
     {
         "kind": "constructor",
         "interface": "FormData",
+        "flavor": "DOM",
```

Comment:
And I guess the origin `FormData` constructor definition is in[`inputfiles/browser.webidl.xml#L4303`](https://github.com/VincentBel/TSJS-lib-generator/blob/18e8e68d6e77f7399ce7f96b5f11da1a2f41e50b/inputfiles/browser.webidl.xml#L4303).